### PR TITLE
Add mechanism to launch containers on a static port

### PR DESCRIPTION
**Pull request by marki1an**

### CODE REVIEW

File: src/test/groovy/org/prebid/server/functional/testcontainers/PbsServiceFactory.groovy [hunk 1]
```diff
@@ -22,10 +24,12 @@ class PbsServiceFactory {
     }
 
     PrebidServerService getService(Map<String, String> config) {
+        final def containerCount = getContainersCount()
```
Comment (hupaloo) [inline src/test/groovy/org/prebid/server/functional/testcontainers/PbsServiceFactory.groovy:27]: 1. No need to declare a var here, it is going to be used only once in that method, inline var pls.
2. Not clear what that method should return from its name: whether it is current container count or max container count, rename pls.
3. In groovy, if method name starts with get* and you are not going to provide parameters in that method call, you can call a method just like that: 'containersCount'

File: src/test/groovy/org/prebid/server/functional/testcontainers/container/PrebidServerContainer.groovy [hunk 2]
```diff
@@ -23,6 +32,7 @@ class PrebidServerContainer extends GenericContainer<PrebidServerContainer> {
     PrebidServerContainer(String dockerImage, Map<String, String> customConfig) {
         super(dockerImage)
         withExposedPorts(PORT, DEBUG_PORT, ADMIN_PORT)
+        withExposedPortsDebug()
```
Comment (hupaloo) [inline src/test/groovy/org/prebid/server/functional/testcontainers/container/PrebidServerContainer.groovy:35]: Words order. You are not exposing ports to debug, you want to expose debugging ports. Rename pls

File: src/test/groovy/org/prebid/server/functional/testcontainers/PbsServiceFactory.groovy [hunk 3]
```diff
@@ -51,4 +53,8 @@ class PbsServiceFactory {
             containers.remove(key)
         }
     }
+
+    private static int containersCount() {
```
Comment (hupaloo) [inline src/test/groovy/org/prebid/server/functional/testcontainers/PbsServiceFactory.groovy:57]: name it like it was named: getContainersCount(). Then you can call that method just like 'containersCount'.

**Review by hupaloo — DISMISSED** [review]
Looks good. Left a couple of minor remarks

**Review by hupaloo — DISMISSED** [review]

**Review by Net-burst — DISMISSED** [review]
LGTM

**Review by SerhiiNahornyi — APPROVED** [review]

**Pull request merged**